<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>1147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>106</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>118</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>572</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Welcome</string>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/icons/icons/akonadi-phone-home.svg</normaloff>:/icons/icons/akonadi-phone-home.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Measure</string>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/icons/icons/dashboard-show.svg</normaloff>:/icons/icons/dashboard-show.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Analyze</string>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/icons/icons/adjustcurves.svg</normaloff>:/icons/icons/adjustcurves.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Configuration</string>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/icons/icons/configure-shortcuts.svg</normaloff>:/icons/icons/configure-shortcuts.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Database</string>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/icons/icons/draw-cuboid.svg</normaloff>:/icons/icons/draw-cuboid.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>111</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>338</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>339</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>308</height>
           </rect>
          </property>
          <attribute name="label">
           <string/>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="razButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(58, 58, 58);
color: rgb(158, 158, 158);
</string>
             </property>
             <property name="text">
              <string>RaZ</string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/icons/window-close.png</normaloff>:/icons/icons/window-close.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="recButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 179, 180);</string>
             </property>
             <property name="text">
              <string>Rec</string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/icons/media-record.png</normaloff>:/icons/icons/media-record.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 127);</string>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/icons/media-playback-stop.png</normaloff>:/icons/icons/media-playback-stop.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="plotButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(67, 76, 255);
background-color: rgb(213, 209, 243);</string>
             </property>
             <property name="text">
              <string>Plot</string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/icons/media-floppy.png</normaloff>:/icons/icons/media-floppy.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(169, 182, 255);
</string>
          </property>
          <widget class="QFrame" name="frame">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>30</y>
             <width>141</width>
             <height>191</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>211</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
border: 1px solid rgb(255, 255, 255);
border-radius: 7px; 
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>7</number>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>42</x>
              <y>60</y>
              <width>87</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
border: 4px solid rgb(0, 0, 0); 
border-radius: 0px; 
</string>
            </property>
            <property name="text">
             <string>Km/h</string>
            </property>
           </widget>
           <widget class="QLCDNumber" name="lcdCount">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>111</width>
              <height>51</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 4px solid rgb(0, 0, 0); 
border-radius: 0px; 
</string>
            </property>
            <property name="digitCount">
             <number>5</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Filled</enum>
            </property>
           </widget>
           <widget class="QLCDNumber" name="lcdSpeed">
            <property name="geometry">
             <rect>
              <x>47</x>
              <y>10</y>
              <width>81</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 4px solid rgb(0, 0, 0); 
border-radius: 0px; 
</string>
            </property>
            <property name="digitCount">
             <number>4</number>
            </property>
            <property name="intValue" stdset="0">
             <number>9999</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>140</y>
              <width>81</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 4px solid rgb(0, 0, 0); 
border-radius: 0px; 
color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Rpm</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1506</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>signal1()</signal>
 </slots>
</ui>
